### PR TITLE
Add Tau programming language.

### PR DESCRIPTION
**Pull request by NicoNex**

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -146,6 +146,20 @@ Did you also built your own version of Monkey? Let me know, just send me an emai
 
 <ul class="list-unstyled pl-2 mt-5 mb-3">
 
+{{< monkey-implementation "NicoNex" "tau" "Go and C"}}
```
Comment (mrnugget) [inline content/_index.md:149]:
```suggestion
{{< monkey-implementation "NicoNex" "tau" "Go and C">}}
```

**Comment by mrnugget** [comment]
Thanks!

**Comment by NicoNex** [comment]
Hi, I've seen that there's an error with the build and the so website didn't update, is there anything wrong with the code I submitted?

**Comment by mrnugget** [comment]
There was a syntax error, but I fixed it and merged. See my commit.

**Comment by mrnugget** [comment]
Ah, wait, you're right

**Pull request merged**